pkg/quota/evaluator/core/persistent_volume_claims.go

```diff
@@ -188,6 +193,11 @@ func (p *pvcEvaluator) Usage(item runtime.Object) (api.ResourceList, error) {
 	if err != nil {
 		return result, err
 	}
+	if pvc.Initializers != nil {
```

Comment:
shouldn't these look more like moving line 204 here, and returning early if uninitialized?

```
result[api.ResourcePersistentVolumeClaims] = resource.MustParse("1")

// if uninitialized, don't compute usage based on any content yet
if pvc.Initializers != nil {
  return result, nil
}
```

also, in other places, you check `pvc.Initializers != nil && len(pvc.Initializers.Pending) == 0`... which is correct?

Reply:
The complex one is correct. I centralized the logic into a utility function in apimachinery.